README.md

```diff
@@ -41,25 +41,25 @@ The relevant benchmarks gathed by QuiVer Benchmarks are defined in [OCR-D's Qual
 
 ## Custom Workflows and Data
 
-The default behaviour of QuiVer Benchmarks is to collect OCR-D's sample Ground Truth workspaces (currently stored in [quiver-data](https://github.com/OCR-D/quiver-data)), executing the [recommended standard workflows](https://ocr-d.de/en/workflows#recommendations) on these and obtaining the relevant [benchmarks](#benchmarks-considered) for each workflow.
+The default behaviour of QuiVer Benchmarks is to collect OCR-D's sample Ground Truth workspaces (currently stored in [quiver-data](https://github.com/OCR-D/quiver-data)), execute the [recommended standard workflows](https://ocr-d.de/en/workflows#recommendations) on these and obtain the relevant [benchmarks](#benchmarks-considered) for each workflow.
 
 You can, however, customize QuiVer Benchmarks to run your own workflows on the sample workspaces or your own OCR-D workspaces.
 
 ### Adding New OCR-D Workflows
 
 Add new OCR-D workflows to the directory `workflows/ocrd_worflows` according to the following conventions:
 
-- OCR workflows have to end with `_ocr.txt`, evaluation workflows with `_eval.txt`. The files will be converted by OtoN to Nextflow files after the container has started.
+- OCR workflows have to end with `_ocr.txt`, evaluation workflows with `_eval.txt`. The files will be converted by [OtoN](https://github.com/MehmedGIT/OtoN_Converter) to Nextflow files after the container has started.
 - workflows have to be TXT files
 - all workflows have to use [`ocrd process`](https://ocr-d.de/en/user_guide#ocrd-process)
 
-You can then either rebuild the Docker image via `docker compose build` or mount the directory to the container via
+You can then either rebuild the Docker image via `docker-compose build` or mount the directory to the container via
```

Comment:
See above.

Reply:
Changed in [27d7e5a](https://github.com/OCR-D/quiver-benchmarks/pull/1/commits/27d7e5ae8cdaf5a8dc817cbfd7be37dfc6d497f5) (for all instances).